### ISSUE TITLE
create new PURL?

### ISSUE THREAD

**Issue by mcourtot** [OBOFoundry/OBOFoundry.github.io#188]
Using the new "PURL" system how would I go about adding a partial redirect from http://purl.obolibrary.org/obo/doid/releases/ to https://github.com/DiseaseOntology/HumanDiseaseOntology/tree/master/releases

Thanks,
Melanie

**Comment by mcourtot** [comment]
I edited and merged the doid.yml (check passed, and how much can I really break it ;)).

Is there a latency time, or else..?

**Comment by cmungall** [comment]
Are you involved in the DO releases? Will you update the docs and the github structure to make it work with the GH release mechanism

An example:
https://github.com/pato-ontology/pato/blob/master/src/ontology/README-editors.md

**Comment by jamesaoverton** [comment]
@mcourtot: Once a change is merged into the `master` branch, Travis has to complete its checks, which usually takes less than 5 minutes. The PURL server checks Travis for updates every ten minutes and rebuilds. So you should see updates about 7 to 17 minutes after merging.

I'll add a note to the README.

**Comment by mcourtot** [comment]
I am not involved in the DO releases. The issue came about discussing with Andra and Elvira as they were using a workaround to resolve the versionIRI PURLs for Wikidata, and I told them that there is a way to setup a partial redirect towards their Github repo, and their release directory.

@cmungall: the doc you pointed to seems to deal mostly with ID conflict and setting a range? The responsible person for DO is Elvira.

@jamesaoverton: thanks! I see there redirection is working now.

**Comment by nlharris** [comment]
Can this be closed?
